exercises/numbers/sumOfRange/sumOfRange.js

```diff
@@ -11,14 +11,24 @@
  * @param {number} rightSummand - The right integer of the range of sum
  * @returns {boolean} The sum of every integer between the two input integers, inclusive
  */
+
 function sumOfRange(leftSummand, rightSummand) {
   // This is your job. :)
+  while (leftSummand < rightSummand) {
```

Comment:
Dunno where this pattern came from, but let's get it out of your repertoire ASAP! It's as if you start for an iterative approach but then decide to do something recursive. Or you had an iterative solution in one place, modified it to be recursive somehow, and then copy + pasted to a new context where it no longer makes sense.

The `while` here is acting like an `if`.

Your program can't handle even moderately large inputs. Get it working correctly first and then go for a fancier solution.

`sumOfRange(1, 50000)` should return `1250025000`, but on my computer it crashes because there are too many levels of recursion.